src/modules/taskStatus.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/extensions */
```

Comment:
Please, don't disable linter checks. Linter checks are placed in order to improve the code quality of students. Kindly remove the linter disable rules. I would recommend fixing the linter errors instead of disabling the linter checks.